### ISSUE TITLE
Question for data collection

### ISSUE THREAD

**Issue by xuyangit** [maxritter/SDC-End-to-end-driving#1]
Good work! I have one question for data collection. Does the speed matters when we collect the training data? Hope you can help me with this. Tks!

**Comment by maxritter** [comment]
Not really. The model learns to do frame-by-frame decisions, so the speed does not have an influence on this.
